modules/dqt/ajax/DeleteDoc.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Hi @kongtiaowang I think @driusan and @xlecours don't want additional ajax files anymore. Instead the request for delete can be done with the RequestHandlerInterface similar to the dqt_setup.class.inc used by the js handleProgressBarSetup function.